.github/workflows/ios-test.yml

```diff
@@ -19,7 +19,7 @@ on:
 jobs:
     test:
         name: Test polyPod iOS app
-        runs-on: macOS-latest
+        runs-on: macOS-12
```

Comment:
GitHub actions should name their runners `latest-we-cared-to-upgrade` or something like that.

Reply:
For some reason, `macOS-latest != macOS-12`...